### PR TITLE
Add "Caramba Switcher.app" latest

### PR DESCRIPTION
**Pull request by toffguy77**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/caramba-switcher.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask "caramba-switcher" do
+  version :latest
+  sha256 :no_check
+
+  # download.caramba.tech was verified as official when first introduced to the cask
+  url "https://download.caramba.tech/apps/switcher/mac/CarambaSwitcherBeta.pkg"
+  name "Caramba Switcher"
+  desc "Experimental, intelligent, self-turning keyboard switcher"
```
Comment (miccal) [inline Casks/caramba-switcher.rb:8]:
```suggestion
  desc "Keyboard switcher"
```

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @toffguy77.  

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

File: Casks/caramba-switcher.rb (hunk 1 above)
Comment (toffguy77) [reply Casks/caramba-switcher.rb:8]: I got this description on official web site. I believe that this is the description the author gave to his project and would like users to read.
If this is the case - I'll change the description.

**Comment by toffguy77** [comment]
> Thank you for your contribution to `homebrew-cask` @toffguy77.
> 
> Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

I did what I could. Please check.

**Comment by miccal** [comment]
> I did what I could. Please check.

Thanks, but none of these files are needed in the `zap` stanza.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @toffguy77.

**Pull request merged**